docs/overview/README.md

```diff
@@ -93,3 +122,94 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 ``` 
+
+## Screenshots
+Below are the screenshots that highlight some of the features of the SkillTree platform. 
+These screenshots broken down into 2 sections:
+- [Dashboard](/overview/#screenshots): a web-based user-friendly interface that allows to create and manage gamified training profile.
+- [Ranking and Progress Display](/overview/#ranking-and-progress-display): JS libraries are integrated into your existing web-based application which provide your users a fully interactive user ranking and progress display.
+
+### Dashboard
+
+A Project is composed of Subjects which are made of Skills and a single skill defines a training unit within the
+gamification framework. Subjects are a way to group and organize skill definitions within a gameified training profile.
+Screenshots below demonstrates dashboard's page to create and manage subjects: 
+![SkillTree Dashboard Subjects](./screenshots/Screenshot_2020-12-16SkillTreeDashboard.png)
+
+Projects are composed of Subjects which are made of Skills and a single skill defines a training unit within the
+gamification framework. To complete a skill, users may need to perform the same action multiple times - repetition is
+important for retention after all. A Skill definition specifies how many times a skill has to be performed. Each
+occurrence is called a Skill Event.
+Screenshots below demonstrates dashboard's management of skill's attribues:
+![SkillsTree Dashboard Edit Skill](./screenshots/Screenshot_2020-12-16SkillTreeDashboard(3).png)
+
+Badges add another facet to the overall gamification profile and allows you to further reward your users by providing
+these prestigious symbols. Badges are a collection of skills and when all of the skills are accomplished that badge is
+earned.
+Screenshots below demonstrates dashboard's page to create and manage badges:
+![SkillsTree Dashboard Badges](./screenshots/Screenshot_2020-12-16SkillTreeDashboard(1).png)
+
+Levels are users' achievement path - the overall goal of the gameified training profile is to encourage users to achieve
+the highest level. Levels are tracked for the entire project as well as for each subject which provides users many ways
+to progress forward.
+Screenshots below demonstrates dashboard's page to manage levels:
+![Skills Platform Overview Image](./screenshots/Screenshot_2020-12-16SkillTreeDashboard(2).png)
+
+Dependencies add another facet to the overall gamification profile, which forces users to complete skills in a specified
+order. If you set up Skill A to depend on the completion of Skill B then no points will be awarded toward Skill A until
+Skill B is fully accomplished.
+Screenshot below demonstrates dashboard's page to manage dependencies:
```

Comment:
"The screenshot below demonstrates management of skill dependencies in the dashboard:"